Bluepill-cli/Bluepill-cli/Bluepill/Bluepill.m

```diff
@@ -425,6 +489,21 @@ - (void)deleteSimulatorWithContext:(BPExecutionContext *)context andStatus:(BPEx
     [context.runner deleteSimulatorWithCompletion:handler.defaultHandlerBlock];
 }
 
+//only called when bp is running in the delete only mode
+- (void)deleteSimulatorOnlyTaskWithContext:(BPExecutionContext *)context {
+    
+    if ([context.runner useSimulatorWithDeviceID: [[NSUUID alloc] initWithUUIDString:context.config.deleteSimUDID]]) {
+        context.simulatorCreated = YES; //if we don't set this flag, deleteSimulatorWithContext() won't proceed
+        
+        NEXT([self deleteSimulatorWithContext:context andStatus:BPExitStatusSimulatorDeleted]);
+        
+    } else {
+        [BPUtils printError:ERROR withString:@"Failed to reconnect simulator"];
+        context.exitStatus = BPExitStatusSimulatorCreationFailed;
```

Comment:
I thought I had added that... if you look at my commit it has it: 
https://github.com/linkedin/bluepill/pull/54/commits/56cdfaf4c31ce5c844c1d679725a6941df13cdd4#diff-1070fc70916a7ce09ff5de4c27a7129bR499